README.md

```diff
@@ -1,14 +1,16 @@
 # businext.ai
 
-**Bus**iness **in**sight **ext**ractor **a**nd **i**nterpreter?
+**Bus**iness **in**sight **ext**ractor **a**nd **i**nterpreter
```

Comment:
Nooooooooooooo!!!!

Reply:
With the demo completed, we now know with certainty that we are indeed a Business Insight Extractor and Interpreter. Therefore the `?` must go.